content/news/038/index.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "This Month in Rust GameDev #38 - September 2022"
+transparent = true
+date = 2022-10-08
+draft = true
++++
+
+<!-- no toc -->
+
+<!-- Check the post with markdownlint-->
+
+Welcome to the 38th issue of the Rust GameDev Workgroup's
+monthly newsletter.
+[Rust] is a systems language pursuing the trifecta:
+safety, concurrency, and speed.
+These goals are well-aligned with game development.
+We hope to build an inviting ecosystem for anyone wishing
+to use Rust in their development process!
+Want to get involved? [Join the Rust GameDev working group!][join]
+
+You can follow the newsletter creation process
+by watching [the coordination issues][coordination].
+Want something mentioned in the next newsletter?
+[Send us a pull request][pr].
+Feel free to send PRs about your own projects!
+
+[Rust]: https://rust-lang.org
+[join]: https://github.com/rust-gamedev/wg#join-the-fun
+[pr]: https://github.com/rust-gamedev/rust-gamedev.github.io
+[coordination]: https://github.com/rust-gamedev/rust-gamedev.github.io/issues?q=label%3Acoordination
+
+- [Announcements](#announcements)
+- [Game Updates](#game-updates)
+- [Learning Material Updates](#learning-material-updates)
+- [Engine Updates](#engine-updates)
```

Comment:
The order of these doesn't match the headings:
```suggestion
- [Engine Updates](#engine-updates)
- [Learning Material Updates](#learning-material-updates)
```
I will submit a PR to fix the template at some point :)